internal/provisioner/fluentbit.go

```diff
@@ -80,12 +79,38 @@ func (f *fluentbit) NewHelmDeployment(logger log.FieldLogger) *helmDeployment {
 	if err != nil {
 		logger.WithError(err).Error("unable to lookup private zone name")
 	}
+	domainNameSplitted := strings.Split(privateDomainName, ".")
+	sumoLogic := `[OUTPUT]
+	Name  forward
+	Match  *
+	Host  example.placeholder.mattermost.com
+	Port  %s
+	tls  On
+	tls.verify  Off
+	#tls.ca_file  /etc/certs/ca-chain.pem`
+
+	env := domainNameSplitted[1]
```

Comment:
This could cause a panic if we don't check the length of the split domain name first. We should do that just in case.